### PR TITLE
Update job specs to be neutral or female coded

### PR DESCRIPTION
**Pull request by lukemorton**
# What 

Update the following roles that were strongly masculine or masculine coded to be neutral or female coded. We did this using a gender decoding tool: https://gender-decoder.katmatfield.com

- Business Analyst
- Senior Business Analyst
- Lead Business Analyst
- Principal Business Analyst
- Product Manager
- Senior Product Manager
- Lead Product Manager
- Senior Data Engineer
- Lead Data Engineer
- Principal UCD
- Principal Technologist

# Why

The tool we use was inspired by a research paper written by Danielle Gaucher, Justin Friesen, and Aaron C. Kay back in 2011, called [Evidence That Gendered Wording in Job Advertisements Exists and Sustains Gender Inequality](https://gender-decoder.katmatfield.com/static/documents/Gaucher-Friesen-Kay-JPSP-Gendered-Wording-in-Job-ads.pdf) (Journal of Personality and Social Psychology, July 2011, Vol 101(1), p109-28).

In this paper the researchers showed job adverts which included different kinds of gender-coded language to men and women and recorded how appealing the jobs seemed and how much the participants felt that they 'belonged' in that occupation. No non-binary people were included in this research, and the research didn't touch on non-binary-coded words.

Their results showed that women felt that job adverts with masculine-coded language were less appealing and that they belonged less in those occupations. For men, feminine-coded adverts were only slightly less appealing and there was no effect on how much the men felt they belonged in those roles.

### CODE REVIEW

**Comment by returnString** [comment]
Looking specifically at the senior and lead eng roles, I feel like they're a bit more blurred than before; I wonder whether there's a more neutral way of implying that we do want people to be specifically capable of _leading_ teams at SFIA 5?

**Comment by returnString** [comment]
Looking specifically at the senior and lead eng roles, I feel like they're a bit more blurred than before; I wonder whether there's a more neutral way of implying that we do want people to be specifically capable of _leading_ teams at SFIA 5?

File: roles/lead_data_engineer.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ UK-based in Birmingham, Bristol, Glasgow, Manchester, Newcastle, London & Swanse
 
 Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
 
-Our Lead Data Engineers enable public sector organisations to embrace a data-driven approach by providing data platforms and services that are high-quality, cost-efficient, and tailored to the client's needs. They lead teams to develop, operate, and maintain these services, as well as ensuring they provide maximum value to data consumers, including analysts, scientists, and business stakeholders.
+Our Lead Data Engineers enable public sector organisations to deliver and run data platforms and services that are high-quality, cost-efficient, and tailored to the client's needs. They support teams to develop, operate, and maintain these services, as well as ensuring they provide maximum value to data consumers, including analysts, scientists, and business stakeholders.
```
Comment (jimstamp) [inline roles/lead_data_engineer.md:7]: "embrace a data-supported approach by providing"?

File: roles/lead_data_engineer.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ UK-based in Birmingham, Bristol, Glasgow, Manchester, Newcastle, London & Swanse
...
 
-As a Lead Data Engineer, you may play one or more roles according to our clients' needs. Sometimes the role is very hands-on and you'll serve as a technical lead for a project, focusing on both delivering engineering work as well as upskilling members of the client team. At other points, you might play more of a technical architect role, ensuring that the wider context of a delivery is considered and maintaining alignment between the operational and analytical aspects of the engineering solution.
+You may play one or more roles according to our clients' needs. Sometimes the role is very hands-on and you'll contribute to a project, focusing on both delivering engineering work as well as upskilling members of the client team. At other points, you might play more of a technical architect role, ensuring that the wider context of a delivery is considered and maintaining alignment between the operational and analytical aspects of the engineering solution.
```
Comment (jimstamp) [inline roles/lead_data_engineer.md:13]: "serve as a technical example to project members"?

File: roles/lead_data_engineer.md (hunk 1 above)
Comment (lukemorton) [reply roles/lead_data_engineer.md:7]: Thanks, will update.

File: roles/lead_data_engineer.md (hunk 2 above)
Comment (lukemorton) [reply roles/lead_data_engineer.md:13]: I've gone with "Sometimes the role is very hands-on and you'll both manage the team and contribute to a project, delivering engineering work including code as well as upskilling members of the client team."

**Pull request merged**